sql/src/main/java/io/crate/planner/TableStatsService.java

```diff
@@ -58,7 +58,7 @@
         DataTypes.STRING);
 
     static final String STMT = "select cast(sum(num_docs) as long), cast(sum(size) as long), schema_name, table_name " +
-                               "from sys.shards group by 2, 3";
+                               "from sys.shards where primary=true group by 3, 4";
```

Comment:
I think we should update the TableStats docs

Reply:
which docs?

Reply:
the javadocs for the TableStats methods

Reply:
The javadoc says: `Returns the number of docs a table has` which is correct, the code was wrong

Reply:
Due to the confusion we had already I think we should be explicit :)